Detections/Box/BoxInactiveUserLogin.yaml

```diff
@@ -0,0 +1,39 @@
+id: edbf38d7-e170-4af2-ad50-1a05b374611b
+name: Box - Inactive user login
+description: |
+  'Detects user login after long inactivity period.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: BoxDataConnector
+    dataTypes:
+      - BoxEvents_CL
+queryFrequency: 1h
+queryPeriod: 1h

```

Comment:
change to 14d as this i the lookback period.

Reply:
fixed